### PR TITLE
Cleaned up docs to focus on runtime.

### PR DESCRIPTION
**Pull request by cloudnautique**
Also added algolia back.

### CODE REVIEW

File: docs/docs/30-installation/02-options.md [hunk 1]
```diff
@@ -29,9 +35,11 @@ Let's Encrypt integration is only useful if you are running a non-local Kubernet
 :::
 
 ## Endpoint domain names
-Acorn provides several installation options for controlling the domain name used to generate endpoints. These are outlined in detail on our [networking page](50-running/02-networking.md#dns).
+
+Acorn provides several installation options for controlling the domain name used to generate endpoints. These are outlined in detail on our [network publishing]<https://docs.acorn.io/production/publish-http>) page.
```
Comment (g-linville) [inline docs/docs/30-installation/02-options.md:39]:
```suggestion
Acorn provides several installation options for controlling the domain name used to generate endpoints. These are outlined in detail on our [network publishing](https://docs.acorn.io/production/publish-http) page.
```

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -106,6 +106,11 @@ const config = {
         darkTheme: darkCodeTheme,
         additionalLanguages: ['cue', 'docker'],
       },
+      algolia: {
+        appId: 'KA4CNV2PUX',
+        apiKey: 'd65a6b6f6eb217f433716ba99aa51bc2',
```
Comment (g-linville) [inline docs/docusaurus.config.js:111]: This seems bad? This is an open source repo.

File: docs/docs/10-home.md [hunk 3]
```diff
@@ -8,13 +8,17 @@ slug: /
 Acorn is a work in progress.  Features will evolve over time and there may be breaking changes between releases.  Please give us your feedback in [Slack](https://slack.acorn.io), [Discussions](https://github.com/acorn-io/runtime/discussions), or [Issues](https://github.com/acorn-io/runtime/issues)!
 :::
 
+:::note
+These are the docs for installing and running Acorn runtime on your own Kubernetes clusters. To learn how to use Acorn to package and deploy your applications, and the Acorn hosted platform please visit [docs.acorn.io](https://docs.acorn.io).
```
Comment (njhale) [inline docs/docs/10-home.md:12]: nit: feels like this flows a little bit better to me; take it or leave it though.

```suggestion
These are the docs for installing and running Acorn runtime on your own Kubernetes clusters. To learn about packaging and deploying your applications using Acorn, or for information about the Acorn hosted platform, please visit [docs.acorn.io](https://docs.acorn.io).
```

File: docs/docs/30-installation/02-options.md (hunk 1 above)
Comment (tylerslaton) [inline docs/docs/30-installation/02-options.md]:
```suggestion
Acorn provides several installation options for controlling the domain name used to generate endpoints. These are outlined in detail on our [network publishing](https://docs.acorn.io/production/publish-http) page.
```

**Review by tylerslaton — APPROVED** [review]

**Review by g-linville — APPROVED** [review]

File: docs/docusaurus.config.js (hunk 2 above)
Comment (cloudnautique) [reply docs/docusaurus.config.js:111]: It's a public key.

**Pull request merged**